pdr_backend/prediction_manager/contracts/PredictionManager.sol

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: None
```

Comment:
It's a bit confusing how PredictionManager & Predictoor relate. And maybe they're more appropriate to be merged? The labels need improving for sure.

What makes this worse is: we already have `have contracts/predictoor_contract.py` & `PredictoorContract` class. However, those labels are inappropriate, and need to be renamed. I just created [#812](https://github.com/oceanprotocol/pdr-backend/issues/812) to fix it. In light of this prediction mgr work, we should do that issue asap.

I recommend: 
1. Do the quick fixes to address my comments (fairly light), then merge this PR. ("Ratchet up" philosophy)
2. Do # 812 (see above)
3. Determine better labels for PredictionManager & Predictoor, OR merge them, OR both
4. Continue the rest of the work on _this_ github issue

Appendix: 
- you should "try on" what happens if you merge Predictoor & PredictionManager. You'll have some Solidity that's slightly redundant for up-vs-down. But then can see everything "flat", which I expect to be much easier for people to read / understand. (Maybe this doesn't make sense too! Your call. But please try it if you think it does)
- Or, if we end up going for better labels, here's a first cut: Predictoor -> OneDirectionFeeds, PredictionManager -> BothDirectionFeeds.

Reply:
> What about Predictoor & PredictionManager -> PredSubmitter & PredSubmitterManager?

This is already much better. Let's use those for now. If we think of something better yet, we can do another search & replace.